data/az_pwsh_cloudinit.cfg

```diff
@@ -0,0 +1,129 @@
+#cloud-config
+password: $pass
+chpasswd: { expire: False }
+ssh_pwauth: True
+
+run_cmd:
+    # Register the Microsoft RedHat repository
+  - [ curl, "https://packages.microsoft.com/config/rhel/7/prod.repo", |, sudo, tee,
+  /etc/yum.repos.d/microsoft.repo ]
+  - yum install -y powershell
```

Comment:
All of these commands should actually be in `[cmd, arg, 'arg']` syntax, its basically coincidence that these commands work without it.